### ISSUE TITLE
Source code is does not safe escape null

### ISSUE THREAD

**Issue by vit1251** [uucidl/uu.xunitgen#8]
You have source code is 

``` python
def quote_attribute(value):
        return value if value is not None else "(null)"
```

and this is very big problem for None bcz it broke XML structure.

If you still use None as marker some thing wrong use "raise Exception"  in another case just use another way.

Now it is sometimes a problem.

**Comment by uucidl** [comment]
Hi,
Can you give an example XML file with the problem?
Thanks

**Comment by Gimpneek** [comment]
Came across this myself when I didn't pass the `package_name` parameter to `write_reports()`. 

It outputs the package attribute on the XML as `package=(null)` instead of `package="(null)"`.

**Comment by uucidl** [comment]
Thanks, that's precise enough to fix :)

**Comment by uucidl** [comment]
Fixed by releasing version 1.0.0 and uploading it to PyPI (The code on master had already been corrected)
